packages/ketcher-core/src/domain/AttachmentPoint.ts

```diff
@@ -102,6 +46,9 @@ export class AttachmentPoint {
     bodyHeight,
     canvas,
     RN,
+    isUsed,
```

Comment:
Can we use more meaningful names instead of `RN`? In addition, I highly encourage you to add more types for variables – there a lot of them with `any` type.